api/app/controllers/spree/api/orders_controller.rb

```diff
@@ -9,6 +9,9 @@ class OrdersController < Spree::Api::BaseController
       class_attribute :admin_order_attributes
       self.admin_order_attributes = [:import, :number, :completed_at, :locked_at, :channel, :user_id, :created_at]
 
+      class_attribute :admin_payment_attributes
+      self.admin_payment_attributes = [:payment_method, :amount, :state, source: [:name, :month, :year, :cc_type, :last_digits]]
```

Comment:
Shouldn’t we allow all attributes a payment source can have? There is more than credit cards. Since this is admin only I am fine with 

```suggestion
      self.admin_payment_attributes = [:payment_method, :amount, :state, source: []]
```

Reply:
@tvdeyen done here https://github.com/solidusio/solidus/pull/3066/files#diff-173268f0a85c59ef13e0753726e16ba8R13, thanks